content/Contributing/Webconsole.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Web Console Log"
+description: "Web Console Log"
+---
+
+## Web Console Log
+
+A web console log can help to diagnose problems with the user interface. This document explains how to save the web console log so it can be submitted for debugging.
+
+ 
+### Firefox
+ 
+
+Open the web console by clicking  <i class="fa fa-bars" aria-hidden="true" title="Menu"></i>&nbsp; **(Menu) > Web Developer > Web Console** or by pressing Ctrl-Shift-K.
+
+In the upper right, set Persist Logs.
```

Comment:
in 78 and higher here is a gear at the right with Persist Logs and Show timestamps.